### DISCUSSION TITLE
安装插件后打开“标签”功能，所有文件就消失了

### DISCUSSION THREAD

**Discussion by luyiling** [MuiseDestiny/zotero-style#317] [Q&A]
安装插件后打开 “标签” 功能，所有文件就消失了；关闭该功能后就能正常显示

**Comment by MuiseDestiny** [comment]
在我优化之前，你可以测试下具体哪个分类下会出现这种情况，会不会有比较特殊的条目。然后最近有时间会优化。

**Reply by scholar-lxy** [reply]
你好，开发者。我使用最新的版本，在zotero里边添加标签，给标签加上颜色后，便签栏李白你不会自动更新出颜色标记的圆圈，需要切换到其他文件夹里在切换回来刷新一下才行。
